.github/linters/.tflint.hcl

```diff
@@ -22,9 +22,9 @@ rule "terraform_naming_convention" {
 rule "terraform_required_version" {
   enabled = true
 }
-rule "terraform_required_providers" {
-  enabled = true
-}
+//rule "terraform_required_providers" {
+//  enabled = true
+//}
```

Comment:
This is intended change?

Reply:
Yes, I would like to leave it here. Related to #14 we must fix it later